throughput/tracker.go

```diff
@@ -51,6 +51,7 @@ func (t *tracker) logState(ctx context.Context, cli *jsonrpc.JSONRPCClient) {
 	var psent int64
 	go func() {
 		defer tick.Stop()
+		prevTime := time.Now()
```

Comment:
can we move `psent` and `prevTime` next to each other to make this easier to follow?